.github/integration-test/evaluate-code.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash -e
 
 INPUT_MEASURE=$1
+DOCKER_COMPOSE_FILE=$2
 BASE_OUTPUT_DIR=$PWD/.github/integration-test/evaluate-code-test
 mkdir "$BASE_OUTPUT_DIR"
 
-docker run -v "$INPUT_MEASURE":/app/measure.json -v "$BASE_OUTPUT_DIR":/app/output/ -e FHIR_SERVER=http://fhir-server:8080/fhir \
-      -e TZ="$(cat /etc/timezone)" --network integration-test_testing-network fhir-data-evaluator
+export FDE_INPUT_MEASURE=$INPUT_MEASURE
```

Comment:
is the INPUT_MEASURE variable needed otherwise? it seems redundant to me to have it here. Same with the output dir.

Reply:
No it is not needed otherwise. That was redundant too